lib/rules/no-descending-specificity/index.js

```diff
@@ -30,7 +30,7 @@ const rule = function(actual) {
 
     root.walkRules(rule => {
       // Ignore custom property set `--foo: {};` and nested property `foo: {};`
-      if (isCustomPropertySet(rule) || isNestedProperty(rule)) {
+      if (isCustomPropertySet(rule) || !isStandardSyntaxRule(rule)) {
```

Comment:
A very minor request; can we have this as a separate statement, please?. This is the convention generally used [else where in the code base](https://github.com/brneto/stylelint/blob/5f1f00ce2d148dd56d0df9bedf16d77ea38072b7/lib/rules/property-no-unknown/index.js#L45-L65)

Reply:
Of course!